domain/src/main/java/org/oppia/android/domain/feedbackreporting/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+"""
+Domain services & definitions corresponding to user feedback reporting.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "feedback_reporting_module",
+    srcs = [
+        "FeedbackReportingModule.kt",
+    ],
+    visibility = ["//:oppia_prod_module_visibility"],
+    deps = [
+        ":dagger",
+        ":report_schema_version",
+    ],
+)
+
+kt_android_library(
+    name = "report_schema_version",
+    srcs = [
+        "ReportSchemaVersion.kt",
+    ],
+    visibility = ["//:oppia_prod_module_visibility"],
```

Comment:
Since this is a dagger constant should this be made private? @BenHenning

Reply:
Is it only meant to be used by the feedbackreporting package @jcqli? If so, then yes. If not, then it should be API visibility since it provides the qualifier to inject whatever property it's corresponding to.

Reply:
I imagine it'll only be used when constructing the report request being sent to the backend so I think this should be private.

Reply:
what do I have to do next @BenHenning

Reply:
I believe if you don't specify visibility at all (remove this line) then it defaults to private.

Reply:
if i didnt specify the visibility, its giving me this error
![Screenshot from 2021-08-20 14-48-52](https://user-images.githubusercontent.com/78534813/130211739-d14ea3c1-59db-4f86-95a8-84410025840b.png)
Instead if i give visibility like( visibility = ["//domain:__pkg__"],) it removes errors and its only visible to domain pkg, Is this correct @jcqli

Reply:
@deepanshu731 I think that's because you added ``"//domain/src/main/java/org/oppia/android/domain/feedbackreporting:report_schema_version",`` to domain/BUILD.bazel. Does it need to be there/does the app build without that dep listed? If so, this can be private. If not, this should be marked as API visibility.

Reply:
thanks @BenHenning , it can be made private